tt_metal/tracy.mk

```diff
@@ -2,6 +2,8 @@ TRACY_LIB = $(LIBDIR)/libtracy.so
 TRACY_INCLUDES = -I$(TT_METAL_HOME)/tt_metal/third_party/tracy/public/tracy/
 TRACY_LDFLAGS = $(LDFLAGS)
 TRACY_DEFINES = -DTRACY_NO_CONTEXT_SWITCH
+TRACY_BUILD_DIR = $(TT_METAL_HOME)/build/profiler/bin
```

Comment:
In CMake, would this be `ExternalProject_Add` with a custom `BUILD_CMD` ? @mo-tenstorrent @TT-billteng

Reply:
yes I think so